kafka/hbase-kafka-proxy/src/main/appended-resources/META-INF/LICENSE.txt

```diff
@@ -0,0 +1,3610 @@
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Code generated by the Protocol Buffer compiler is owned by the owner
+of the input file used when generating it.  This code is not
+standalone and requires a support library to be linked with it.  This
+support library is itself covered by the above license.
+
+----
+This product includes JSch licensed under the Revised BSD.
+
+Copyright (c) 2002-2015 Atsuhiko Yamanaka, JCraft,Inc.
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+  1. Redistributions of source code must retain the above copyright notice,
+     this list of conditions and the following disclaimer.
+
+  2. Redistributions in binary form must reproduce the above copyright
+     notice, this list of conditions and the following disclaimer in
+     the documentation and/or other materials provided with the distribution.
+
+  3. The names of the authors may not be used to endorse or promote products
+     derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESSED OR IMPLIED WARRANTIES,
+INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL JCRAFT,
+INC. OR ANY CONTRIBUTORS TO THIS SOFTWARE BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA,
+OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+----
+This product includes jersey-client, jersey-core, jersey-json, jersey-server, jersey-guice,
+JAXB RI, javax.annotation API, Java Servlet API, JavaServer Pages(TM) API, javax.ws.rs-api,
+JAXB API bundle for GlassFish V3, Expression Language 3.0, HK2 API module, ServiceLocator,
+HK2 Implementation Utilities, OSGi resource locator bundle - used by various API
+providers that rely on META-INF/services mechanism to locate providers, aopalliance version
+1.0 repackaged as a module, javax.inject:1 as OSGi bundle,JSP implementation,
+jersey-repackaged-guava, jersey-container-servlet-core, jersey-core-client,
+jersey-core-common, jersey-core-server, jersey-media-jaxb, under the
+license COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.1
+The GNU General Public License (GPL) Version 2 with CLASSPATH EXCEPTION
```

Comment:
Why is this here? are the above things dual licensed? I think so? If so, we should just say "under CDDL v1.1" and leave out the GPL stuff. we choose which license we redistribute under, and as an ASF project we can't choose GPL.